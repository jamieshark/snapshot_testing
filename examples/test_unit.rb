require "bundler/setup"
require "test/unit"
require "snapshot_testing/test_unit"

class ExampleTest < Test::Unit::TestCase
  include SnapshotTesting::TestUnit

  def test_snapshot
    assert_snapshot "hello"
    assert_snapshot "goodbye"
  end

  def test_named_snapshot
    assert_snapshot "named.unit.txt", "named"
  end

  define_method "test_escapes_regex_[]" do
    assert_snapshot "hello"
  end
end
