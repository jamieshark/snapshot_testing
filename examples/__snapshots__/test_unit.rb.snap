snapshots["test_snapshot 1"] = <<-SNAP
hello
SNAP

snapshots["test_snapshot 2"] = <<-SNAP
goodbye
SNAP

snapshots["test_escapes_regex_[] 1"] = <<-SNAP
hello
SNAP
